### ISSUE TITLE
Associations used by analysts for segregation not reflected in "associations" list

### ISSUE THREAD

**Issue by lfkennedy** [spawaskar-cora/cora-docs#394]
The list of associations that analysts can use to link elements do not include OSR/regression information or age.  In sequences with MNI >1, we have been segregating out these separate individuals using this information.  Adding these lines of evidence may be worthwhile, especially if the end result of this process will be a reflection of the segregation reports that we generate in the lab.

![image](https://user-images.githubusercontent.com/34662044/53583894-1b1bc680-3b48-11e9-8d7d-368b5d209949.png)

**Comment by CTegtmeyer** [comment]
@clegarde curious if you think we still need to add this with the enhancements we've made to the segregation report dashboard.

**Comment by clegarde** [comment]
I agree that this is outdated and can be closed. We're still fine tuning the "association of elements report" but that issue is captured otherwise.
